test/ttl-to-json-test.js

```diff
@@ -83,14 +59,16 @@ describe("ttl-to-json", function() {
           });
       });
 
-      it("should return an error if --ttlfile given non-existent file", function(done) {
-          var file =  os.tmpdir() + "/foobar" + Math.random() + ".ttl"
+      it("should return an error if given non-existent ttl file", function(done) {
+          var file =  os.tmpdir() + "/wikiwiki" + Math.random() + ".ttl"
           var cmd = ttlToJsonPath + " --ttlfile " + file;
-          exec(cmd, function (error, stdout, stderr) {
-              error.code.should.equal(1);
-              stderr.should.contain("File " + file + " does not exist!");
-              done();
-          });
+          commonTest.cmdFileNonexistent(cmd, file, done);
+      });
+
+      it("should return an error if ttlfile file is not readable", function(done) {
+          var file = os.tmpdir() + '/wikiwiki' + Math.random() + ".ttl"
```

Comment:
The file is not being created here - just the file path so we can use it in building command line immediately below.  The file is actually created and deleted in the commonTest.cmdFileUnreadable call after that.   To make this more clear, I'll rename this filepath where I use this.